charts/asserts/large.yaml

```diff
@@ -0,0 +1,36 @@
+server:
+  resources:
+    limits:
+      memory: 3Gi
+    requests:
+      cpu: 1
+      memory: 3Gi
+
+redisgraph:
+  master:
+    resources:
+      limits:
+        memory: 1Gi
+      requests:
+        cpu: 1
+        memory: 1Gi
+
+tsdb:
+  server:
+    retentionPeriod: 90d
```

Comment:
What is this `90d` retention period?

Reply:
The tsdb retention time. Should we make it longer? Also, I can make it match the graph.

Reply:
I've added the retention to match the graph. We will have to do some re-work to have a single global value, but this is ok for now.

Reply:
90 days sounds fine. I was curious why it differs from the 30 days you mentioned earlier.

Reply:
I made the examples 90days bc it's more realistic for an actual deployment. Default values are mostly to just get up and running.